src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -710,46 +710,51 @@ def _flatten_tuples(
             head_state.add_edge(src_node, None, nsdfg_node, connector, memlet)
 
         def make_temps(
-            output_data: gtir_builtin_translators.FieldopData,
+            inner_data: gtir_builtin_translators.FieldopData,
         ) -> gtir_builtin_translators.FieldopData:
             """
             This function will be called while traversing the result of the lambda
             dataflow to setup the intermediate data nodes in the parent SDFG and
             the data edges from the nested-SDFG output connectors.
```

Comment:
I do not like the description:
- It is a bit cryptic
- It does not impose any restriction on `inner_data` (it must be part of a nested SDFG).
- What happens if the translation is not needed, i.e. `inner_data` is already global.

Also the name sounds a bit strange, it is super general, but the function is actually super specific as it only works on nested SDFGs.
How about renaming it to `create_output_for_nested_sdfg()` or `map_nested_data_container_to_parent_sdfg()` or `map_data_from_nested_sdfg()`?

An alternative description would be:
This function makes a data container, denoted by `inner_data`, that lives inside a nested SDFG available in the parent SDFG.
For this the data container inside the nested SDFG will be marked as global (a requirement of the SDFG IR) and create a new data container within the parent SDFG, with the same properties (shape, stride, etc.) appropriate remapped using the symbol mapping table.
If `inner_data` was already mapped into the parent SDFG then [I do not know what happens then, but I guess that thing is returned?}